src/lib/seed-db.js

```diff
@@ -12,7 +12,7 @@ module.exports = function * (config) {
 function * setupHoldAccount () {
   const holdAccount = yield models.Account.findByName('hold')
   if (!holdAccount) {
-    yield models.Account.create({name: 'hold', balance: '0'})
+    yield models.Account.create({name: 'hold', minimum_allowed_balance: '0', balance: '0'})
```

Comment:
I think both amounts are supposed to be numbers, but it looks like it will work either way.